docarray/index/backends/helper.py

```diff
@@ -21,13 +21,23 @@ def inner(self, *args, **kwargs):
 
 
 def _execute_find_and_filter_query(
-    doc_index: BaseDocIndex, query: List[Tuple[str, Dict]]
+    doc_index: BaseDocIndex, query: List[Tuple[str, Dict]], reverse_order: bool = False
 ) -> FindResult:
     """
     Executes all find calls from query first using `doc_index.find()`,
     and filtering queries after that using DocArray's `filter_docs()`.
 
     Text search is not supported.
+
+    Args:
+        doc_index: Document index instance.
+            Either InMemoryExactNNIndex or HnswDocumentIndex.
+        query: Dictionary containing search and filtering configuration.
+        reverse_order: Flag indicating whether to sort in descending order. If set to
+            False (default), the sorting will be in ascending order.
+
+    Returns:
+        Sorted documents and their corresponding scores.
```

Comment:
this is not the right style for docstring

Reply:
I got too used to langchain docstrings lol